<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyjobWidget</class>
 <widget class="QWidget" name="CopyjobWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copy objects from file</string>
  </property>
  <property name="windowIcon">
   <iconset theme="edit-copy">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="rowWrapPolicy">
    <enum>QFormLayout::DontWrapRows</enum>
   </property>
   <property name="labelAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="formAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="actionLabel">
     <property name="text">
      <string>Action</string>
     </property>
     <property name="buddy">
      <cstring>actionCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="actionCombo">
     <item>
      <property name="text">
       <string>Copy selected object</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Copy object with path</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="formatLabel">
     <property name="text">
      <string>Format</string>
     </property>
     <property name="buddy">
      <cstring>formatCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="formatCombo">
     <item>
      <property name="text">
       <string>Object name lists</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Regular expressions</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="dstpathLabel">
     <property name="text">
      <string>Destination</string>
     </property>
     <property name="buddy">
      <cstring>dstpathButton</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="dstpathLayout">
     <item>
      <widget class="QLineEdit" name="dstpathEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Select destination path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="dstpathButton">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="srcpathLabel">
     <property name="text">
      <string>List source</string>
     </property>
     <property name="buddy">
      <cstring>srcpathButton</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="srcpathLayout">
     <item>
      <widget class="QLineEdit" name="srcpathEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Select source file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="srcpathButton">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="columnLabel">
     <property name="text">
      <string>Column</string>
     </property>
     <property name="buddy">
      <cstring>columnSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="columnSpin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="levelLabel">
     <property name="text">
      <string>Level</string>
     </property>
     <property name="buddy">
      <cstring>levelSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="levelSpin">
     <property name="specialValueText">
      <string comment="level">Any</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>actionCombo</tabstop>
  <tabstop>formatCombo</tabstop>
  <tabstop>dstpathEdit</tabstop>
  <tabstop>dstpathButton</tabstop>
  <tabstop>srcpathEdit</tabstop>
  <tabstop>srcpathButton</tabstop>
  <tabstop>columnSpin</tabstop>
  <tabstop>levelSpin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
